src/couch/src/couch_auth_lockout.erl

```diff
@@ -0,0 +1,65 @@
+% Licensed under the Apache License, Version 2.0 (the "License"); you may not
+% use this file except in compliance with the License.  You may obtain a copy of
+% the License at
+%
+%   http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+% WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+% License for the specific language governing permissions and limitations under
+% the License.
+
+-module(couch_auth_lockout).
+-include_lib("couch/include/couch_db.hrl").
+
+-define(LRU, couch_lockout_lru).
+
+-export([is_locked_out/3, lockout/3]).
+
+is_locked_out(#httpd{} = Req, UserName, UserSalt) ->
+    case config:get_boolean("couch_lockout", "enable", true) of
```

Comment:
@rnewson 

Would it make sense now (after renaming to `couch_auth_lockout.erl`) to put it under the `[chttpd_auth]` section as `lockout` key?

Reply:
Ah no, maybe not at the moment, because it isn't a cluster wide setting / functionality yet ...

Reply:
@rnewson Since Jan gives a +1 for this, it should be moved to `[chttpd_auth]` with key `lockout`.